ui/apps/platform/src/Containers/Vulnerabilities/components/AdvancedFiltersToolbar.tsx

```diff
@@ -63,7 +63,8 @@ type AdvancedFiltersToolbarProps = {
     onFilterChange: (searchFilter: SearchFilter, payload: OnSearchPayload) => void;
     className?: string;
     defaultFilters?: DefaultFilters;
-    includeCveFilters?: boolean;
+    includeCveSeverityFilters?: boolean;
+    includeCveStatusFilters?: boolean;
```

Comment:
This property needs to be split since Platform CVE does not show the Severity filter dropdown, but _does_ show the Status dropdown.